src/popup/popup.html

```diff
@@ -82,38 +82,75 @@ <h5 class="b-header margin-bottom-normal">Use</h5>
         <div class="section-filter__license margin-top-big" data-storage-key-name="licenseFilter">
           <h5 class="b-header margin-bottom-normal">Licenses</h5>
           <input type="checkbox" id="CC0" />
-          <label for="CC0"> <i class="icon cc-zero has-background-white margin-right-smaller"></i> CC0</label><br />
+          <label for="CC0"
+            ><i class="icon cc-logo has-background-white margin-right-smaller"></i>
+            <i class="icon cc-zero has-background-white margin-right-smaller"></i> CC0
+            <div id="cc0" class="tooltip" style="margin-left: 150px ;">
+              <i id="cc0" class="icon question-color question-circle"></i
+              ><span id="cc0-gen" class="tooltiptext tooltip-right"></span></div></label
+          ><br />
           <input type="checkbox" id="PDM" />
-          <label for="PDM"><i class="icon cc-pd has-background-white margin-right-smaller"></i>Public Domain Mark</label
+          <label for="PDM">
+            <i class="icon cc-logo has-background-white margin-right-smaller"></i
+            ><i class="icon cc-pd has-background-white margin-right-smaller"></i>Public Domain Mark
+            <div class="tooltip" style="margin-left: 48px ;">
+              <i id="pdm" class="icon question-color question-circle"></i
+              ><span id="pdm-gen" class="tooltiptext tooltip-right"></span></div></label
```

Comment:
Putting the tooltip inside the `label` has a unwanted consequence. Clicking the tooltip will check/uncheck the checkbox. This is not what we want.